web/webapi.py

```diff
@@ -401,7 +401,11 @@ class cgiFieldStorage(cgi.FieldStorage):
     to incorrect encoding in Python 3.
     """
 
-    def make_file(self):
+    def make_file(self, binary=None):
```

Comment:
If it is a flag then it is __True__ or __False__.

Reply:
Okay, I copied the style from Python 2's cgi. Yet, I can change the default value to `True`, as that follows what we are doing in the body of the method.

Reply:
Oh.  OK.  Then leave it as is.